app/models/OkcBlotter/booking.rb

```diff
@@ -0,0 +1,4 @@
+class OkcBlotter::Booking < ApplicationRecord
+  belongs_to :offense, class_name: 'OkcBlotter::Offense', foreign_key: 'offense_id'
+  validates  :transient, :inmate_number, :booking_number, :booking_date, presence: true
```

Comment:
The namespace looks good!
You have the associations backwards though...
PDF has many Bookings has many Offenses
Offenses belongs to Booking belongs to PDF
so the bookings table has pdf_id
and the offenses table has booking_id